first-party/jib-layer-filter-extension-maven/src/main/java/com/google/cloud/tools/jib/maven/extension/layerfilter/Configuration.java

```diff
@@ -56,7 +57,22 @@ public String getToLayer() {
 
   private List<Filter> filters = new ArrayList<>();
 
+  /**
+   * Whether to create separate layers for dependencies that stem from the parent POM. The parent
+   * layers are created after the filters have been applied. For every original layer and every
+   * layer defined by the filters, a parent layer will be created (if not empty).
+   */
+  private boolean createParentLayers;
+
   public List<Filter> getFilters() {
     return filters;
   }
+
+  public boolean isCreateParentLayers() {
+    return createParentLayers;
+  }
+
+  public void setCreateParentLayers(boolean parentLayers) {
```

Comment:
Ok. In the test I now mock the getter instead of using the getter. So, it is simliar to the approach with getFilters.
I also use the global config instance in all tests now. For this, I make sure to set the Mock to defaults in setUp.  (since my tests don't mock getFilters and they shall not be influenced by the other tests that do so)